routemaster-client.gemspec

```diff
@@ -16,7 +16,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^spec/})
   spec.require_paths = %w(.)
 
+  spec.add_runtime_dependency     'typhoeus', '~> 1'
```

Comment:
I think that `~> 1` will be expanded as: `>= 1.0.0`, which allows bumps to `2.0.0` or `3.1.2`.
http://guides.rubygems.org/patterns/#pessimistic-version-constraint

What about using `~> 1.1`?